lib/content-services/src/lib/common/services/nodes-api.service.ts

```diff
@@ -239,4 +239,25 @@ export class NodesApiService {
 
         return new NodeMetadata(metadata, nodeEntry.entry.nodeType);
     }
+
+    /**
+     * Gets the list of holds assigned to the node.
+     *
+     * @param nodeId ID of the target node
+     * @param options Optional parameters supported by JS-API
+     * @returns List of assigned holds Observable<Hold[]>
+     */
+    getNodeAssignHolds(nodeId: string, options?: any): Observable<Hold[]> {
```

Comment:
We should not use any in this case, possible options to be passed should be strictly defined as they are for `listParents` endpoint

Reply:
added 
```
 {
            includeSource?: boolean;
        } & NodesIncludeQuery &
            ContentPagingQuery
```